buffer-netty/src/main/java/io/micronaut/buffer/netty/ByteBufAllocatorConfiguration.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.buffer.netty;
+
+import io.micronaut.core.annotation.Nullable;
+
+/**
+ * Interface for the Netty bytebuf allocator configuration.
+ *
+ * @since 3.3.0
+ * @author graemerocher
+ */
+public interface ByteBufAllocatorConfiguration {
+    String DEFAULT_ALLOCATOR = "netty.default.allocator";
```

Comment:
What is the value of `default` in the key? Are there other allocators that are not default?